liblouis/lou_backTranslateString.c

```diff
@@ -1535,7 +1515,7 @@ back_passDoAction(const TranslationTableHeader *table, int *pos, int mode,
 	while (passIC < currentRule->dotslen) switch (passInstructions[passIC]) {
 		case pass_string:
 		case pass_dots:
-			if ((output->length + passInstructions[passIC + 1]) > output->maxlength)
+			if (((unsigned int)output->length + passInstructions[passIC + 1]) > (unsigned int)output->maxlength)
```

Comment:
Used type casting here because changing the structure introduced other miss matches, and I was aiming to fix them :)